lib/reporters/cli/index.js

```diff
@@ -315,6 +358,42 @@ _.assignIn(PostmanCLIReporter, {
             hAlign: 'left'
         }]);
 
+        timings.waitAverage && summaryTable.push([{
```

Comment:
Most of the code in this section is duplicated, can be compacted with a list and a loop 😄